### PR TITLE
Add the ability to use custom audio button message

### PR DESCRIPTION
**Pull request by Lmenkovaa**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -38,8 +38,11 @@
   },
   "dependencies": {
     "@babel/runtime-corejs3": "^7.14.7",
-    "@material-ui/icons": "4.9.1",
-    "audio-recorder-polyfill": "^0.4.1"
+    "@material-ui/icons": "^4.11.2",
+    "audio-recorder-polyfill": "^0.4.1",
+    "chat-ui-react": "^0.2.14",
```
Comment (KovalM) [inline package.json]: Why we need old version dependency?

File: src/mui/MuiMessage.tsx [hunk 2]
```diff
@@ -1,8 +1,17 @@
...
 import React from 'react';
 
 import { Message, MessageContent } from '../chat-types';
 
+export function AudioMessageButton(props: any): JSX.Element {
+  if (props.component) {
+    return props.component;
+  }
+  return <Button
+  style= {{flex: '1 0 0%', display: 'flex',flexDirection: 'row', border: '0px', color: 'rgb(63, 81, 181)', margin: '0px 0px 5px 10px', height: '20px', width: '20px', boxShadow: '1px 1px 2px rgba(0,0,0,.3)', marginTop:'auto'}}
```
Comment (KovalM) [inline src/mui/MuiMessage.tsx]: Move styles to start of component

File: src/mui/MuiMessage.tsx [hunk 3]
```diff
@@ -82,6 +100,18 @@ export function MuiMessage({
...
             )}
+            {message.type === 'text_audio' && (
+              <div style={{flex: '0 1 0%', display: 'flex', flexDirection: 'row'}}>
+                <Typography
+                  variant="body1"
+                  style={{ overflowWrap: 'break-word', whiteSpace: 'pre-wrap' }}>
+                    {message.content}
+                </Typography>
+                <div onClick={playSoundContent}>
+                  <AudioMessageButton {...audioButtonProps}/>
```
Comment (KovalM) [inline src/mui/MuiMessage.tsx:132]:
```suggestion
                  <AudioMessageButton onClick={playSoundContent} {...audioButtonProps}/>
```

File: src/mui/MuiMessage.tsx [hunk 4]
```diff
@@ -82,6 +100,18 @@ export function MuiMessage({
...
+                </div>
```
Comment (KovalM) [inline src/mui/MuiMessage.tsx:133]:
```suggestion
```

File: src/mui/MuiMessage.tsx (hunk 3 above)
Comment (KovalM) [reply src/mui/MuiMessage.tsx:132]: I don't test it, try please

**Review by KovalM — CHANGES_REQUESTED** [review]
We need to add dist files to repo for getting this lib by including github. See example here: https://github.com/KovalM/chat-ui-react/tree/github-publication

File: src/mui/MuiMessage.tsx (hunk 3 above)
Comment (Lmenkovaa) [reply src/mui/MuiMessage.tsx:132]: It's doesn't work

**Review by KovalM — APPROVED** [review]

**Pull request merged**